index.js

```diff
@@ -8,20 +8,21 @@
 'use strict'
 
 const prepack = require('prepack')
-const utils = require('rollup-pluginutils')
 
 module.exports = function rollupPluginPrepack (options) {
-  options = Object.assign({ include: '**/*.js' }, options)
-
-  const filter = utils.createFilter(options.include, options.exclude)
```

Comment:
Why removing the filtering?

Reply:
> Why removing the filtering?

Because the is not supported in the last verion, check out the options https://prepack.io/getting-started.html

Reply:
What you mean by not supported. It needs javascript file, so we need to filter. But the above comments defeat this discussions since we will require to pass dist file as in your repo.

Reply:
Are `include` and `exclude` part of Prepack options?